message-index/messages/GHC-1629/example1/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Too many type arguments in constructor pattern
+---
+
```

Comment:
An explanation of this specific instance of the error would be useful (i.e. the @Bool is unnecessary)

Reply:
[6441dd6](https://github.com/haskell/error-messages/pull/123/commits/6441dd6dbcd90920b1a25830921a39c16eedc479)